src/test/java/lotto/validate/NumberValidationTest.java

```diff
@@ -0,0 +1,37 @@
+package lotto.validate;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class NumberValidationTest {
+
+    private NumberValidation numberValidation;
+
+    @BeforeEach
+    void setUp() {
+        numberValidation = new NumberValidation();
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3, 4, 5, 6})
+    void 로또_번호_유효성_검사_성공(int number) {
+        // given
+        // when
+        numberValidation.checkNumberRange(number);
+        // then
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 46, 47, 48, 49, 50})
+    void 로또_번호_유효성_검사_실패(int number) {
+        // given
+        // when
+        // then
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            numberValidation.checkNumberRange(number);
+        }).withMessageMatching("로또 번호는 1부터 45까지의 숫자만 가능합니다.");
```

Comment:
```suggestion
        }).withMessageMatching("로또 번호는 1부터 45까지의 숫자만 가능합니다.");
```
미션에서는 로또 범위가 변경되지 않지만,
만약 변경된다면 불필요하게 테스트가 실패될수 있지 않을까? 라는 생각을 해볼수도 있을것 같아요!

`일반화`하거나 `변하는 부분과 변하지 않는 부분 분리`같은 방법들이 있을것 같아요.
일반화 한다면, `로또번호는 숫자만 가능합니다` 이런식으로요!